src/main/java/org/craftercms/studio/api/v2/service/audit/AuditService.java

```diff
@@ -93,17 +66,18 @@ List<AuditLog> getAuditLog(String siteId, String siteName, int offset, int limit
      * @param clusterNodeId filter logs by given cluster node id
      * @return number of audit log entries
      */
-    int getAuditLogTotal(String siteId, String siteName, String user, List<String> operations,
+    int getAuditLogTotal(String siteId, String user, List<String> operations,
                                     boolean includeParameters, ZonedDateTime dateFrom, ZonedDateTime dateTo,
-                                    String target, String origin, String clusterNodeId);
+                                    String target, String origin, String clusterNodeId) throws SiteNotFoundException;
 
     /**
      * Get audit log entry by id
      *
      * @param auditLogId audit log id
+     * @param siteId site ID
```

Comment:
Perhaps we also document that this can be `null` which means all sites and admin activities.

Reply:
Updated on other methods retrieving filtered (by site) log entry lists